buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -94,25 +82,23 @@ public void apply(Project project) {
 
         // After each task we determine if there are clusters that are no longer needed.
         configureStopClustersHook(project);
-
-        // Since we have everything modeled in the DSL, add all the required dependencies e.x. the distribution to the
-        // configuration so the user doesn't have to repeat this.
-        autoConfigureClusterDependencies(project, rootProject, container);
-    }
-
-    private static File getExtractDir(Project project) {
-        return new File(project.getRootProject().getBuildDir(), "testclusters/extract/");
     }
 
     private NamedDomainObjectContainer<ElasticsearchCluster> createTestClustersContainerExtension(Project project) {
+
+        project.getPlugins().apply(DistributionDownloadPlugin.class);
```

Comment:
I'd move this up to the first line of the plugin `apply()` method. It makes it very obvious that this other plugin is a "dependency" of this plugin.